powerbi-docs/developer/visuals/create-bar-chart.md

```diff
@@ -586,61 +573,107 @@ In addition to scaling, this update method also handles selections and colors. T
 
 ### Populate the properties pane
 
-The final method in the `IVisual` function is [`enumerateObjectInstances`](visual-api.md#enumerateobjectinstances-optional). This method goes through all the *"objects"* in the *capabilities.json* file (in our case, `enableAxis` and `colorSelector`) and places them inside the [**Format** pane](conditional-format.md). The object's name is available from `EnumerateVisualObjectInstancesOptions`.
+The final method in the `IVisual` function is [`getFormattingModel`](visual-api.md#getFormattingModel-optional). This method builds and return modern format pane formatting model object that contain all format pane components and properties, Then places it inside the [**Format** pane](format-pane.md).
```

Comment:
'returns'

Reply:
@mberdugo I think Mona will fix my English mistakes as in here